web/src/beta/features/Editor/useLayers.ts

```diff
@@ -52,17 +52,13 @@ export default function ({ sceneId, isVisualizerReady, visualizerRef }: LayerPro
   const handleLayerSelect = useCallback(
     (layerId?: string) => {
       if (!isVisualizerReady) return;
+      layerId ? setSelectedLayerId({ layerId }) : setSelectedLayerId(undefined);
```

Comment:
```suggestion
setSelectedLayerId(layerId ? { layerId } : undefined);
```

Reply:
I'm not very sure about the detail of select states between editor & core right now. 
But i think it's good to make the logic on editor part more straight forward & make sure there's no obvious bugs for now, and maybe we can have a deep look into the selections and do some refactors later together with core when members are free from view3.0 project.